controllers/grid/eventLog/EventLogGridCellProvider.php

```diff
@@ -87,7 +87,7 @@ public function getTemplateVarsFromRowColumn($row, $column)
                             if (isset($params['reviewAssignmentId'])) {
                                 $reviewAssignment = $reviewAssignmentDao->getById($params['reviewAssignmentId']);
                                 if ($reviewAssignment && $reviewAssignment->getReviewMethod() === ReviewAssignment::SUBMISSION_REVIEW_METHOD_OPEN) {
-                                    $userName = $reviewAssignment->getUserFullName();
+                                    $userName = $element->getUserFullName();
```

Comment:
@Vitaliy-1, can you review this change?

Reply:
Oops, forgot to remove deprecated methods, working on it: https://github.com/pkp/pkp-lib/issues/9067